instrumentation/mybatis/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/mybatis/MybatisTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.mybatis;
+
+import static org.mockito.Mockito.when;
+
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.instrumentation.testing.junit.AgentInstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import org.apache.ibatis.binding.MapperMethod;
+import org.apache.ibatis.mapping.MappedStatement;
+import org.apache.ibatis.mapping.SqlCommandType;
+import org.apache.ibatis.reflection.factory.DefaultObjectFactory;
+import org.apache.ibatis.session.Configuration;
+import org.apache.ibatis.session.defaults.DefaultSqlSession;
+import org.junit.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.mockito.Mockito;
+
+class MybatisTest {
+
+  @RegisterExtension
+  protected static final InstrumentationExtension testing = AgentInstrumentationExtension.create();
+
+  private static final String SPAN_NAME =
+      "io.opentelemetry.javaagent.instrumentation.mybatis.RecordMapper.updateRecord";
```

Comment:
Would it be better if span name does not include the full package name, `RecordMapper.updateRecord`

Reply:
> Would it be better if span name does not include the full package name, `RecordMapper.updateRecord`

For the MyBatis scenario, including the fully qualified name can better find the corresponding interface. Just like the database instrumentation, directly span name is the query statement, which will make the whole thing clearer. Such as following figure.
![image](https://github.com/open-telemetry/opentelemetry-java-instrumentation/assets/25423978/be066357-51fb-47ea-883a-660911ffb44c)